thrust/thrust/functional.h

```diff
@@ -190,13 +190,7 @@ struct CCCL_DEPRECATED binary_function
  *  \see https://en.cppreference.com/w/cpp/utility/functional/plus
  *  \see binary_function
  */
-template <typename T = void>
-struct plus : public ::cuda::std::plus<T>
-{
-  using first_argument_type _LIBCUDACXX_DEPRECATED_IN_CXX11  = T;
-  using second_argument_type _LIBCUDACXX_DEPRECATED_IN_CXX11 = T;
-  using result_type _LIBCUDACXX_DEPRECATED_IN_CXX11          = T;
-}; // end plus
+using ::cuda::std::plus;
```

Comment:
I wonder whether we should drop the documentation as well (and whether it actually builds and shows correctly)

Reply:
Yeah I am also in favor of just dropping it.